### PR TITLE
Subject importer

### PR DESCRIPTION
**Pull request by marten**
This is based on the script used to load up Wildcam, but with tests and with ids as parameters. This way we can just kick off an import from Jenkins.

### CODE REVIEW

**Comment by edpaget** [comment]
LGTM

File: app/workers/subject_import_worker.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
+require 'csv'
+
+class SubjectImportWorker
+  include Sidekiq::Worker
```
Comment (camallen) [inline app/workers/subject_import_worker.rb]: should we push this to a data queue, `sidekiq_options queue: :data_high`

**Comment by camallen** [comment]
LGTM too

File: app/workers/subject_import_worker.rb (hunk 1 above)
Comment (marten) [inline app/workers/subject_import_worker.rb]: Are the queues live yet? Or will the normal worker pick them up for now?

Comment (camallen) [inline app/workers/subject_import_worker.rb]: not yet - adam's setting them up and no normal worker won't pick them up. so up to you about specifying the queue.

Comment (marten) [inline app/workers/subject_import_worker.rb]: Merging it now, and telling slackbot to remind me to move the queue next week

Comment (camallen) [inline app/workers/subject_import_worker.rb]: cool

**Pull request merged**